frontend/components/Course/CourseSidebar.tsx

```diff
@@ -184,6 +184,7 @@ const CourseSidebar = (props: CourseSidebarProps) => {
             >
                 <a
                     href="https://airtable.com/shrIZxIjyAE3gOUSg"
+                    rel="noreferrer"
```

Comment:
Just curious why we added this?

Reply:
Sorry I missed this comment - it was a lint warning, and to be honest it should probably be a lint error because it is a security vulnerability. See here for more details: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md